ui/app/AppLayouts/Profile/views/AppearanceView.qml

```diff
@@ -13,25 +12,21 @@ import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 import StatusQ.Controls 0.1 as StatusQ
 
-import "../popups"
-import "../stores"
-
 SettingsContentBase {
     id: appearanceView
 
+    required property var localSettings
+
     function updateTheme(theme) {
-        localAppSettings.theme = theme
+        localSettings.theme = theme
         Theme.changeTheme(theme)
```

Comment:
when propagating requests via signals we will have all `Theme.changeTheme` calls in AppMain.